packages/block-library/src/columns/block.json

```
@@ -28,7 +28,9 @@
 			}
 		},
 		"spacing": {
-			"blockGap": true,
+			"blockGap": {
+				"default": "2em"
```

How about an __experimental property?

```suggestion
				"__experimentalDefault": "2em"
```

---

Good catch @adamziel, It looks like a new API would be a more controversial choice in the context of the WP 6.0 release. Regardless of how we apply that to `wp/6.0` branch, this PR will require documentation changes as a follow-up.

---

Thanks, yes that's a great idea Adam, I'll update to add an `experimental` prefix 👍

---

I've added the `__experimental` prefix in: https://github.com/WordPress/gutenberg/pull/41098/commits/7774b7ce4bba13eeace06d8306fcf7a5f7e4539e

If this PR lands in time, @ramonjd and I are happy to write up documentation changes in a follow-up.